admin/server.js

```diff
@@ -31,6 +31,7 @@ app.use(cookieParser());
 app.use(session({
   secret: 'sojo sut ed oterces le'
 }));
+app.use(test)
```

Comment:
What is the purpose of this?

Reply:
weird I don't remember adding it. It's not suppose to be there. Let me fix it.